testsuite/classlibrary/TestRelease.sc

```diff
@@ -0,0 +1,33 @@
+TestRelease : UnitTest {
```

Comment:
Name for this class should be `TestNode` I think, and the other class `TestNode_Server`, so that the component functionality being tested is clear. This also fits what I recommend in my guide. That also allows you to remove `node` from the names of the tests in this class.